utils/keyUtils.js

```diff
@@ -0,0 +1,21 @@
+const base58 = require('bs58');
+const { InvalidPublicKeyError } = require('../service/errors');
+
+const validatePublicKey = publicKey => {
+    if(!publicKey){
+        throw new InvalidPublicKeyError("Public key not supplied.")
+    }
+    let pk;
+    try {
+        pk = base58.decode(publicKey);
+    } catch (err) {
+        throw new InvalidPublicKeyError("Public key must be provided in base 58.")
+    }
+    if(pk.length !== 32){
```

Comment:
Rather crude check on only the length, but given the registrar will be replaced soon I guess it is okay.

Reply:
Yes, ideally we would have a check here to see whether it's on the curve. I couldn't seem to find a simple JS package that does public key validation. Good to keep in mind here that, for now, if an invalid public key is used, the VCs won't verify.